realtime/server.js

```diff
@@ -195,7 +195,7 @@ socket.on('connection', function (client) {
 
         delete people[client.id];
         delete usernames[username];
-        socket.sockets.emit('part', username);
+        socket.sockets.emit('part', {username: username, deletedMessageid: deletedMessageid});
```

Comment:
nit: style, when your object contains more than 2 keys expand it in multiple lines like
```
{
    username: username,
    password: password,
}
```